dfdaemon/server.go

```diff
@@ -117,7 +117,9 @@ func NewFromConfig(cfg config.Properties) (*Server, error) {
 
 // Start runs dfdaemon's http server.
 func (s *Server) Start() error {
-	proxy.WithDirectHandler(handler.New())(s.proxy)
+	if err := proxy.WithDirectHandler(handler.New())(s.proxy); err != nil {
```

Comment:
[proxy.WithDirectHandler](https://github.com/dragonflyoss/Dragonfly/blob/master/dfdaemon/proxy/proxy.go#L81) always returns `nil`. Maybe we can just leave it be? Or explicitly use 
```_ = proxy.WithDirectHandler(handler.New())(s.proxy)```
 to show that we've ignored the error.